packages/styles/scss/components/combo-button/_combo-button.scss

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright IBM Corp. 2023
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../config' as *;
+@use '../../motion' as *;
+@use '../../utilities/convert' as *;
+
+/// Combo Button styles
+/// @access public
+/// @group combo-button
+@mixin combo-button {
+  .#{$prefix}--combo-button__container {
+    display: inline-flex;
+    column-gap: rem(1px);
+  }
+
+  $triggerSizes: (
+    'sm': rem(32px),
+    'md': rem(40px),
+    'lg': rem(48px),
+  );
```

Comment:
Yet another example of why we need https://github.com/carbon-design-system/carbon/discussions/10619! 😄